Source/WebCore/page/EventHandler.cpp

```diff
@@ -1706,7 +1700,7 @@ std::optional<Cursor> EventHandler::selectCursor(const HitTestResult& result, bo
 #if ENABLE(CURSOR_VISIBILITY)
 void EventHandler::startAutoHideCursorTimer()
 {
-    Page* page = m_frame.page();
+    Page* page = m_frame->page();
```

Comment:
Why not use checkedPage here as well?

Reply:
The changes to EventHandler are not meant to be exhaustive. This patch's focus is RenderElement. I just made some changes to EventHandler store it in a CheckedPtr/CheckedRef and changing m_frame ended up making the change large.

Looking at this particular function though, you can see the page is only used to query a setting. I figure this was safe so I didn't opportunistically use CheckedPtr for the page here.